client/src/pages/MoviesList/index.js

```diff
@@ -1,18 +1,59 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 
 import { MovieListBackground } from '../../styles/Background'
 import Marquee from '../../components/Marquee'
 import Posters from './Posters'
 import SelectedMovie from './SelectedMovie'
+import TurnsTable from './TurnsTable'
+import { useHouses } from '../../providers/Houses'
+import { useTurns } from './utility'
+import { ReelImage } from '../../styles/LoadingReel'
 
 const MoviesList = () => {
   const [selectedMovie, setSelectedMovie] = useState()
 
+  const { currentHouse, userId } = useHouses()
+  const { getHouseTurns, houseTurns } = useTurns()
+
+  useEffect(() => {
+    if (currentHouse && !houseTurns) {
+      getHouseTurns(currentHouse.id)
+    }
+  }, [getHouseTurns, houseTurns, currentHouse])
+
+  if (!currentHouse || !houseTurns) {
```

Comment:
Be careful about this kind of condition. You're inferring that because there is no currentHouse or no houseTurns that they must be loading. But if either request failed and there was an error, this would still be true and we would just be showing a spinner indefinitely. 
I'd suggest adding `error` and `loading` states to your hooks and using those values to determine when to show a loader.

```
const {isLoading, data} = myHook()

if(isLoading) {
    return <Loading/>
}

if(data) {
    return <PosterStack>
}


```